shlr/www/p/vendors/dagre.core.js

```diff
@@ -1038,7 +1038,7 @@ function normalizeEdge(g, e) {
   g.removeEdge(e);
 
   var dummy, attrs, i;
-  for (i = 0, ++vRank; vRank < wRank; ++i, ++vRank) {
+  for (i = 0, ++vRank; vRank < wRank; i++, vRank++) {
```

Comment:
This is autogenerated code

Reply:
ok 2258c7faa95fc2c19e1fee59d3d1ffe8cc8297cd